README.md

```diff
@@ -35,7 +35,7 @@ Retrieve Pedestal and switch to a publicly-available version:
 
     git clone https://github.com/pedestal/pedestal.git pedestal
     cd pedestal
-    git checkout 0.5.1
+    git checkout 0.5.3
```

Comment:
hasn't 0.5.4 been cut?

Reply:
Yes, this PR predates that. I can update.